src/components /forecastSummary.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import PropTypes from "prop-types";
+import WeatherIcon from "react-icons-weather";
+import moment from "moment";
+
+function ForecastSummary(props) {
+  const { date, description, icon, temperature, onSelect } = props;
```

Comment:
Can destructure the props directly again here:
```suggestion
function ForecastSummary({ date, description, icon, temperature, onSelect }) {
```